### ISSUE TITLE
api.v3.blockchir 

### ISSUE THREAD

**Issue by CoinMarketCapAustraliaNFTs** [Blockchair/Blockchair.Support#930]
Benefits
Build
Enjoy access to JetBrains tools as it provides a broader set of IDE for different programming languages such as Python, Java, and Kotlin.

Use Bootstrap Studio to create responsive websites using the Bootstrap framework.

Polypane cares about making your site responsive, fast and accessible, including over 80 accessibility tests, 14 different emulators and two dozen tools.

Launch
GitHub Pages allows you to launch websites for you and your projects. Hosted directly from your GitHub repository. Just edit, push, and your changes are live.

Digital Ocean provides simple cloud hosting, built for developers.

Learn
Have fun learning how to code with TwilioQuest, an educational video game designed to teach a new generation of developers how to change the world with code.

Learn while building through Microsoft Azure’s Cloud Advocates Web Development for Beginners course. This 12-week, 24-lesson curriculum teaches you all about JavaScript, CSS, and HTML basics.

Level up on trending coding skills at your own pace with Educative’s interactive, text-based courses.

Offers and additional benefits sses, blocks and embedded text data...

Home
Privacy policy
TL;DR: Blockchair does not collect personal data or share it with third parties. We don't track you.

server {
    server_name blockchair.com;
    access_log off;
Why is this important?
One of the key advantages of cryptocurrencies is that they enable (pseudo)anonymous transactions. In most cases the user’s address and transaction details are made public and cannot be deleted, but their personal identity remains unknown if no link exists between the user and their blockchain data.

Privacy is at risk when you share any information with third parties. Cryptocurrency exchanges with KYC policies, online retailers that require delivery addresses and web wallets associated with phone numbers all require you to share information.

What’s more, most web servers maintain default logs of your IP address and User Agent (browser name and operating system), the dates and times of your browsing activity and, most importantly, the URLs you visited. Ordinarily, a cryptocurrency address page is only visited by the address owner, while the transaction page is visited by the transaction parties. Blockchain explorers can therefore easily trace the digital fingerprint that links addresses and transactions. Unfortunately, this data is also picked up by the web analytics tools (Google Analytics, Baidu Tongji, Yandex.Metrica), advertising platforms and similar third-party services.

User data can be traced in others ways too. CDN providers like Cloudflare, Incapsula and AWS Shield act as reverse proxies, which means some websites require you to request data from a CDN in order to use the site. You therefore share your information with the provider.

In addition to these data tracking services, there are several other ways how users can be identified online.

HTTP referer: a client request header that allows a server to trace the previous site you visited. Say you visit example.com followed by explorer.com/1YourBitcoinAddress then the former will receive information that you have come from the latter;
Web beacon (bug): an invisible web page element that confirms a user has visited a web page. This is used to collect user analytics;
Cookies: user activity data stored in the user’s browser. Third-party cookies can also be embedded in the site’s code (if it contains elements from other sites);
Evercookie: a JavaScript app that stores zombie cookies on a computer. These cookies are extremely difficult to remove since Evercookie recreates them whenever they are deleted;
Device / browser fingerprint: the device and browser information collected for user identification;
Browser extensions.
Why is it unsafe to share you personal data?
Most blockchain explorers and cryptocurrency companies store user information, including available balances, lists of transactions and types of cryptocurrency.

They might sell this information, publish it, share it with government agencies, or they might be hacked. If it becomes public knowledge that you have significant funds stored in cryptocurrency, you’re likely to be targeted by cyber criminals. Your personal safety may be at risk too.

Why is Blockchair the safer option?
When you connect to Blockchair your browser automatically sends us information about your computer, User Agent, IP address, and the page you want to visit. Since this data may expose your identity, we do not permanently store information about you;
We do not use cookies that can be used to identify you. We may only set our own cookies to improve your user experience and help us to fight botnets and spammers. See below for details;
Your browser won’t send HTTP referer headers when leaving Blockchair.com. This means you can move to other sites without your browsing activity being traced by those sites;
We do not use CDN-providers, including those used to distribute JavaScript libraries and styles. We do not use any third-party site elements, web analytics tools (such as Google Analytics) and hit counters. Therefore, other parties do not receive information about you.
We do not collect any user data, neither on ours nor third-party servers in our “Blockchair” Chrome Extension.
What data do we store and how do we use this data?
We only collect anonymous aggregated data that allows us to improve our website features. We count visitors, analyze popular searches, cryptocurrencies, sortings and other queries.

We also store the incoming IP addresses in masked or clear form for short periods of 1 to 2 days. This is to limit the rate of API requests.

Your device may store first-party cookies, such as those that keep the night mode on, store referer information, unique visitor and session ID.

Collected data is used to improve user experience and compile website traffic statistics. Session data is deleted on a regular basis.

We can also store and use the data you provide us through email or with other channels of communication such as Telegram or contact forms located on our website. We can use the stored data to provide you with our services, to defend our legal rights, to improve accuracy, effectiveness, security, usability of our products or we can use it in any other way you will agree to.

Overall, we won't keep your data longer than necessary.

Please, be informed, that this Privacy Policy applies on all Blockchair products and has force only in a part where it doesn't contradict to Terms of Service of any product of Blockchair Limited. This Privacy Policy provides a general legal framework for the data collection in Blockchair products, hence the respective Terms of Service for each Blockchair product can regulate this issue differently within its respective scope of applicability. In case of contradiction between provisions of any Blockchair Terms of Service and this Privacy Policy, the provisions of Blockchair Terms of Service in question should prevail.

Privacy Policy updates
We will publish any updates to our Privacy Policy at this page (https://blockchair.com/privacy) and in the GitHub repository at https://github.com/Blockchair/Blockchair.Support/blob/master/PRIVACY.md plus the link to the updated version will be available at the bottom of all our site pages.

MINTER-BLOCKCHAIR blocks and embedded text data...

Home
Privacy policy
TL;DR: Blockchair does not collect personal data or share it with third parties. We don't track you.

server {
    server_name blockchair.com;
    access_log off;
Why is this important?
One of the key advantages of cryptocurrencies is that they enable (pseudo)anonymous transactions. In most cases the user’s address and transaction details are made public and cannot be deleted, but their personal identity remains unknown if no link exists between the user and their blockchain data.

Privacy is at risk when you share any information with third parties. Cryptocurrency exchanges with KYC policies, online retailers that require delivery addresses and web wallets associated with phone numbers all require you to share information.

What’s more, most web servers maintain default logs of your IP address and User Agent (browser name and operating system), the dates and times of your browsing activity and, most importantly, the URLs you visited. Ordinarily, a cryptocurrency address page is only visited by the address owner, while the transaction page is visited by the transaction parties. Blockchain explorers can therefore easily trace the digital fingerprint that links addresses and transactions. Unfortunately, this data is also picked up by the web analytics tools (Google Analytics, Baidu Tongji, Yandex.Metrica), advertising platforms and similar third-party services.

User data can be traced in others ways too. CDN providers like Cloudflare, Incapsula and AWS Shield act as reverse proxies, which means some websites require you to request data from a CDN in order to use the site. You therefore share your information with the provider.

In addition to these data tracking services, there are several other ways how users can be identified online.

HTTP referer: a client request header that allows a server to trace the previous site you visited. Say you visit example.com followed by explorer.com/1YourBitcoinAddress then the former will receive information that you have come from the latter;
Web beacon (bug): an invisible web page element that confirms a user has visited a web page. This is used to collect user analytics;
Cookies: user activity data stored in the user’s browser. Third-party cookies can also be embedded in the site’s code (if it contains elements from other sites);
Evercookie: a JavaScript app that stores zombie cookies on a computer. These cookies are extremely difficult to remove since Evercookie recreates them whenever they are deleted;
Device / browser fingerprint: the device and browser information collected for user identification;
Browser extensions.
Why is it unsafe to share you personal data?
Most blockchain explorers and cryptocurrency companies store user information, including available balances, lists of transactions and types of cryptocurrency.

They might sell this information, publish it, share it with government agencies, or they might be hacked. If it becomes public knowledge that you have significant funds stored in cryptocurrency, you’re likely to be targeted by cyber criminals. Your personal safety may be at risk too.

Why is Blockchair the safer option?
When you connect to Blockchair your browser automatically sends us information about your computer, User Agent, IP address, and the page you want to visit. Since this data may expose your identity, we do not permanently store information about you;
We do not use cookies that can be used to identify you. We may only set our own cookies to improve your user experience and help us to fight botnets and spammers. See below for details;
Your browser won’t send HTTP referer headers when leaving Blockchair.com. This means you can move to other sites without your browsing activity being traced by those sites;
We do not use CDN-providers, including those used to distribute JavaScript libraries and styles. We do not use any third-party site elements, web analytics tools (such as Google Analytics) and hit counters. Therefore, other parties do not receive information about you.
We do not collect any user data, neither on ours nor third-party servers in our “Blockchair” Chrome Extension.
What data do we store and how do we use this data?
We only collect anonymous aggregated data that allows us to improve our website features. We count visitors, analyze popular searches, cryptocurrencies, sortings and other queries.

We also store the incoming IP addresses in masked or clear form for short periods of 1 to 2 days. This is to limit the rate of API requests.

Your device may store first-party cookies, such as those that keep the night mode on, store referer information, unique visitor and session ID.

Collected data is used to improve user experience and compile website traffic statistics. Session data is deleted on a regular basis.

We can also store and use the data you provide us through email or with other channels of communication such as Telegram or contact forms located on our website. We can use the stored data to provide you with our services, to defend our legal rights, to improve accuracy, effectiveness, security, usability of our products or we can use it in any other way you will agree to.

Overall, we won't keep your data longer than necessary.

Please, be informed, that this Privacy Policy applies on all Blockchair products and has force only in a part where it doesn't contradict to Terms of Service of any product of Blockchair Limited. This Privacy Policy provides a general legal framework for the data collection in Blockchair products, hence the respective Terms of Service for each Blockchair product can regulate this issue differently within its respective scope of applicability. In case of contradiction between provisions of any Blockchair Terms of Service and this Privacy Policy, the provisions of Blockchair Terms of Service in question should prevail.

Privacy Policy updates
We will publish any updates to our Privacy Policy at this page (https://blockchair.com/privacy) and in the GitHub repository at https://github.com/Blockchair/Blockchair.Support/blob/master/PRIVACY.md plus the link to the updated version will be available at the bottom of all our site pages.

Contacts
Please share your comments and suggestions at <info@blockchair.com>.
Contacts
Please share your comments and suggestions at <danielleahy304@gmail.com> 
Status of our services
General status: All services are functioning normally.
Last check: 2022-07-07T21:12:10Z
API requests processed today: 13454883, requests per second on average: 176.27

✅️ Web interface is ok
✅️ Application programming interface is ok
✅️ Onion v3 interface is ok
-------------------------------------
✅️ Bitcoin explorer is ok
✅️ Bitcoin Cash explorer is ok
✅️ Ethereum explorer is ok
✅️ Litecoin explorer is ok
✅️ Bitcoin SV explorer is ok
✅️ Dogecoin explorer is ok
✅️ Dash explorer is ok
✅️ Ripple explorer is ok
✅️ Groestlcoin explorer is ok
✅️ Bitcoin Testnet explorer is ok
✅️ Stellar explorer is ok
✅️ Monero explorer is ok
✅️ Cardano explorer is ok
✅️ Zcash explorer is ok
✅️ Mixin explorer is ok
✅️ EOS explorer is ok
✅️ Ethereum Testnet explorer is ok
✅️ eCash explorer is ok
✅️ Polkadot explorer is ok
✅️ Kusama explorer is ok
⚠️ Solana explorer may be lagging behind
© 2022 Blockchair / About / Contact.
